frontend/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM docker.io/library/node:20-bookworm-slim AS build
+
+WORKDIR /app
+
+COPY ./yarn.lock /app/yarn.lock
+COPY ./package.json /app/package.json
+COPY ./dashboard/package.json /app/dashboard/package.json
+COPY ./relay-workflows-lib /app/relay-workflows-lib
+COPY ./workflows-lib /app/workflows-lib
+RUN yarn install --frozen-lockfile
+
+COPY . .
+WORKDIR /app/relay-workflows-lib
+RUN yarn run relay
```

Typically we don't want to change the `WORKDIR` frequently
```suggestion
RUN yarn run --cwd relay-workflows-lib relay
```

---

Oh, I was not aware of the `--cwd` flag in yarn. That is much better.

---

Need to use `--cwd=relay-workflows-lib` to get this to work